tests/v2/actions/charts/charts.go

```diff
@@ -11,6 +11,7 @@ const (
 	serverURLSettingID   = "server-url"
 	rancherChartsName    = "rancher-charts"
 	rancherPartnerCharts = "rancher-partner-charts"
+	stackStateChart      = "suse-observability"
```

Comment:
we have another constant for suse-observability in actions/charts/stackstate.go. can we remove this and make use of that as its a public constant?